### PR TITLE
fix: adding Platform Support and ML Framework Support sections in the README; fixing some typos in the README

### PR DESCRIPTION
**Pull request by bhaoz**
*Issue #, if available:*
N/A

*Description of changes:*
fix: adding Platform Support and ML Framework Support sections in the README; fixing some typos in the README

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by adheshgarg — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -30,6 +33,15 @@ The SageMaker HyperPod CLI is a tool that helps submit training jobs to the Amaz
   - Or you can follow the [Readme under helm_chart folder](https://github.com/aws/sagemaker-hyperpod-cli/blob/main/helm_chart/readme.md) to install Kubeflow Training Operator.
 - Configure [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) to point to the correct region where your HyperPod clusters are located. 
 
+## Platform Support
+
+SageMaker HyperPod CLI currently supports Linux and MacOS platforms. Windows platform support is working in progress
```
Comment (jswudi) [inline README.md:38]: let's say Windows platform is not supported now. We don't have ECD yet.

File: README.md [hunk 2]
```diff
@@ -30,6 +33,15 @@ The SageMaker HyperPod CLI is a tool that helps submit training jobs to the Amaz
...
+
+## ML Framework Support
+
+SageMaker HyperPod CLI currently supports start training job with:
+- PyTorch ML Framework. Version requirements: PyTorch > 1.10
```
Comment (jswudi) [inline README.md:43]: should it be > 1.0.0 ?

File: README.md (hunk 1 above)
Comment (bhaoz) [reply README.md:38]: Sure, let me change that

File: README.md (hunk 2 above)
Comment (bhaoz) [reply README.md:43]: It should be > 1.10 - https://discuss.pytorch.org/t/how-to-install-torchrun/135138

**Pull request closed without merging**